### ISSUE TITLE
Gas Boiler temperature limited to 37 degrees

### ISSUE THREAD

**Issue by dbartelmus** [oischinger/ha_vicare#139]
Hi there,
this is nice to have actively developed Viessmann library for HomeAssistant.

Is there a reason why:
VICARE_TEMP_HEATING_MAX = 37
is set for 37 degrees while gas boilers can reach up to 82 degrees according to Viessmann API (and boiler control panel it self)?

**Comment by crazyfx1** [comment]
That's the value for the maximum target temperature for the room temperature. In the ViCare app the maximum is even 30°C.

**Comment by dbartelmus** [comment]
Thank your for your answer.

Entity added via ViCare integration to home assistant is controlling radiator flow temperature - this is different from target temperature for the room temperature. This setting should consider heating curve and outdoor temperature for optimal gas efficiency, but this is not the main topic of this issue. Most often default temperature is 45-50°C for condensing boiler due to so called "dew point".

> "Target temperature for the room temperature"
most often is controlled via external thermostat which turns on and off gas boiler and boiler flow pump. 

> "In the ViCare app the maximum is even 30°C."
in my case 82°C is the ViCare maximum settings and 20°C is the lowest, im using gas boiler BOKA 050W 25KW

**Comment by crazyfx1** [comment]
You are right, the min / max temperatures should be read dynamically and not be hardcoded.

**Comment by oischinger** [comment]
something like this should work, right?
https://github.com/oischinger/ha_vicare/pull/151

**Comment by dbartelmus** [comment]
Thank you for this commit, i will test it in next week
